libs/wingc/src/type_check.rs

```diff
@@ -207,6 +207,24 @@ pub struct Interface {
 	pub env: SymbolEnv,
 }
 
+impl Interface {
+	fn is_resource(&self) -> bool {
+		// TODO: This should check that the interface extends `IResource` from
+		// the SDK, not just any interface with the name `IResource`
```

Comment:
The way to do this it to lookup the `cloud.IResource` name in the std lib namespace. And then compare the pointers (since there should be only one such interface imported). But I'm fine with making this a future task.

Reply:
Yeah, I wasn't sure how to do a lookup from here. One option might be to move this method so it exists on the `Types` struct instead, but that feels a bit weird, so I was also thinking of simply adding `is_resource` as a `bool` field on `Interface`